en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect.md

```diff
@@ -0,0 +1,296 @@
+# Deploying a GraphQL API in Choreo Connect
+
+GraphQL is a querying language for APIs. Initially it was developed by Facebook. When compared to the REST APIs, GraphQL APIs allow
+you to query only the required data. Response will have only the data you requested (over-fetching and under-fetching will not happen). Using a single request you can fetch data
+related to multiple resources. This is not possible with REST APIs. If REST API is used, you will have to consider multiple requests
+to gather the same amount of data. GraphQL APIs are based on a strong type system. Schema Definition Language (SDL) file defines the
+contract between client and the server. It explains how client can access the data provided by the server. This page explains how to
+deploy a GraphQL API in Choero Connect and also explains the supported features of Choreo Connect for GraphQL APIs.
+
+!!! attention "Update Level _"
```

Comment:
need to update the version in here.

Reply:
We should add the date of the update as well.